src/components/modal.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  // const userConfirm = () => {
+  //   ageConfirmStore.setUserOfAge('over21')
+  //   setUserConfirmed(true);
+  // }
+  //
+  // const underAge = () => {
+  //   window.open("https://www.cdc.gov/alcohol/fact-sheets/underage-drinking.htm");
+  // }
```

Comment:
Would be best to refactor this in or remove it

Reply:
Deleted out and maybe refactor at some other point.